diff -Naur a/src/client/linux/crash_generation/crash_generation_client.cc b/src/client/linux/crash_generation/crash_generation_client.cc
--- a/src/client/linux/crash_generation/crash_generation_client.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/crash_generation/crash_generation_client.cc	2023-06-20 10:56:18.746685403 -0700
@@ -36,7 +36,7 @@
 
 #include "common/linux/eintr_wrapper.h"
 #include "common/linux/ignore_ret.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 
diff -Naur a/src/client/linux/handler/exception_handler.cc b/src/client/linux/handler/exception_handler.cc
--- a/src/client/linux/handler/exception_handler.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/handler/exception_handler.cc	2023-06-20 10:56:18.750685408 -0700
@@ -94,7 +94,7 @@
 #include "client/linux/minidump_writer/linux_dumper.h"
 #include "client/linux/minidump_writer/minidump_writer.h"
 #include "common/linux/eintr_wrapper.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 #if defined(__ANDROID__)
 #include "linux/sched.h"
diff -Naur a/src/client/linux/handler/exception_handler_unittest.cc b/src/client/linux/handler/exception_handler_unittest.cc
--- a/src/client/linux/handler/exception_handler_unittest.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/handler/exception_handler_unittest.cc	2023-06-20 10:56:18.750685408 -0700
@@ -49,7 +49,7 @@
 #include "common/linux/linux_libc_support.h"
 #include "common/tests/auto_tempdir.h"
 #include "common/using_std_string.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 #include "google_breakpad/processor/minidump.h"
 
 using namespace google_breakpad;
diff -Naur a/src/client/linux/log/log.cc b/src/client/linux/log/log.cc
--- a/src/client/linux/log/log.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/log/log.cc	2023-06-20 10:56:18.754685413 -0700
@@ -32,7 +32,7 @@
 #include <android/log.h>
 #include <dlfcn.h>
 #else
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 #endif
 
 namespace logger {
diff -Naur a/src/client/linux/minidump_writer/cpu_set.h b/src/client/linux/minidump_writer/cpu_set.h
--- a/src/client/linux/minidump_writer/cpu_set.h	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/cpu_set.h	2023-06-20 10:56:21.690688837 -0700
@@ -34,7 +34,7 @@
 #include <string.h>
 
 #include "common/linux/linux_libc_support.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 
diff -Naur a/src/client/linux/minidump_writer/directory_reader.h b/src/client/linux/minidump_writer/directory_reader.h
--- a/src/client/linux/minidump_writer/directory_reader.h	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/directory_reader.h	2023-06-20 10:56:21.694688842 -0700
@@ -37,7 +37,7 @@
 #include <string.h>
 
 #include "common/linux/linux_libc_support.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 
diff -Naur a/src/client/linux/minidump_writer/line_reader.h b/src/client/linux/minidump_writer/line_reader.h
--- a/src/client/linux/minidump_writer/line_reader.h	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/line_reader.h	2023-06-20 10:56:21.694688842 -0700
@@ -34,7 +34,7 @@
 #include <string.h>
 
 #include "common/linux/linux_libc_support.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 
diff -Naur a/src/client/linux/minidump_writer/linux_dumper.cc b/src/client/linux/minidump_writer/linux_dumper.cc
--- a/src/client/linux/minidump_writer/linux_dumper.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/linux_dumper.cc	2023-06-20 10:56:18.766685426 -0700
@@ -50,7 +50,7 @@
 #include "common/linux/memory_mapped_file.h"
 #include "common/linux/safe_readlink.h"
 #include "google_breakpad/common/minidump_exception_linux.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 using google_breakpad::elf::FileID;
 
diff -Naur a/src/client/linux/minidump_writer/linux_dumper_unittest_helper.cc b/src/client/linux/minidump_writer/linux_dumper_unittest_helper.cc
--- a/src/client/linux/minidump_writer/linux_dumper_unittest_helper.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/linux_dumper_unittest_helper.cc	2023-06-20 10:56:18.766685426 -0700
@@ -38,7 +38,7 @@
 #include <unistd.h>
 
 #include "common/scoped_ptr.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 #if defined(__ARM_EABI__)
 #define TID_PTR_REGISTER "r3"
diff -Naur a/src/client/linux/minidump_writer/linux_ptrace_dumper.cc b/src/client/linux/minidump_writer/linux_ptrace_dumper.cc
--- a/src/client/linux/minidump_writer/linux_ptrace_dumper.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/linux_ptrace_dumper.cc	2023-06-20 10:56:18.766685426 -0700
@@ -56,7 +56,7 @@
 #include "client/linux/minidump_writer/directory_reader.h"
 #include "client/linux/minidump_writer/line_reader.h"
 #include "common/linux/linux_libc_support.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 // Suspends a thread by attaching to it.
 static bool SuspendThread(pid_t pid) {
diff -Naur a/src/client/linux/minidump_writer/minidump_writer.cc b/src/client/linux/minidump_writer/minidump_writer.cc
--- a/src/client/linux/minidump_writer/minidump_writer.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/minidump_writer.cc	2023-06-20 10:56:18.770685431 -0700
@@ -78,7 +78,7 @@
 #include "common/linux/linux_libc_support.h"
 #include "common/minidump_type_helper.h"
 #include "google_breakpad/common/minidump_format.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace {
 
diff -Naur a/src/client/linux/minidump_writer/proc_cpuinfo_reader.h b/src/client/linux/minidump_writer/proc_cpuinfo_reader.h
--- a/src/client/linux/minidump_writer/proc_cpuinfo_reader.h	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/linux/minidump_writer/proc_cpuinfo_reader.h	2023-06-20 10:56:21.702688851 -0700
@@ -35,7 +35,7 @@
 
 #include "client/linux/minidump_writer/line_reader.h"
 #include "common/linux/linux_libc_support.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 
diff -Naur a/src/client/minidump_file_writer.cc b/src/client/minidump_file_writer.cc
--- a/src/client/minidump_file_writer.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/client/minidump_file_writer.cc	2023-06-20 10:56:18.794685460 -0700
@@ -40,7 +40,7 @@
 #include "common/linux/linux_libc_support.h"
 #include "common/string_conversion.h"
 #if defined(__linux__) && __linux__
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 #endif
 
 #if defined(__ANDROID__)
diff -Naur a/src/common/linux/file_id.cc b/src/common/linux/file_id.cc
--- a/src/common/linux/file_id.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/common/linux/file_id.cc	2023-06-20 10:56:18.846685520 -0700
@@ -45,7 +45,7 @@
 #include "common/linux/linux_libc_support.h"
 #include "common/linux/memory_mapped_file.h"
 #include "common/using_std_string.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 namespace elf {
diff -Naur a/src/common/linux/memory_mapped_file.cc b/src/common/linux/memory_mapped_file.cc
--- a/src/common/linux/memory_mapped_file.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/common/linux/memory_mapped_file.cc	2023-06-20 10:56:18.854685530 -0700
@@ -39,7 +39,7 @@
 #include <unistd.h>
 
 #include "common/memory_range.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 
diff -Naur a/src/common/linux/safe_readlink.cc b/src/common/linux/safe_readlink.cc
--- a/src/common/linux/safe_readlink.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/common/linux/safe_readlink.cc	2023-06-20 10:56:18.858685533 -0700
@@ -31,7 +31,7 @@
 
 #include <stddef.h>
 
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace google_breakpad {
 
diff -Naur a/src/common/memory_allocator.h b/src/common/memory_allocator.h
--- a/src/common/memory_allocator.h	2023-01-27 13:36:21.000000000 -0700
+++ b/src/common/memory_allocator.h	2023-06-20 10:56:21.818688987 -0700
@@ -46,7 +46,7 @@
 #define sys_munmap munmap
 #define MAP_ANONYMOUS MAP_ANON
 #else
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 #endif
 
 namespace google_breakpad {
diff -Naur a/src/processor/testdata/linux_test_app.cc b/src/processor/testdata/linux_test_app.cc
--- a/src/processor/testdata/linux_test_app.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/processor/testdata/linux_test_app.cc	2023-06-20 10:56:18.990685688 -0700
@@ -45,7 +45,7 @@
 #include <string>
 
 #include "client/linux/handler/exception_handler.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 
 namespace {
 
diff -Naur a/src/tools/linux/md2core/minidump-2-core.cc b/src/tools/linux/md2core/minidump-2-core.cc
--- a/src/tools/linux/md2core/minidump-2-core.cc	2023-01-27 13:36:21.000000000 -0700
+++ b/src/tools/linux/md2core/minidump-2-core.cc	2023-06-20 10:56:18.994685693 -0700
@@ -51,7 +51,7 @@
 #include "common/using_std_string.h"
 #include "google_breakpad/common/breakpad_types.h"
 #include "google_breakpad/common/minidump_format.h"
-#include "third_party/lss/linux_syscall_support.h"
+#include <linux_syscall_support.h>
 #include "tools/linux/md2core/minidump_memory_range.h"
 
 #if ULONG_MAX == 0xffffffffffffffff
