--- a/Makefile.in	2023-01-27 13:36:21.000000000 -0700
+++ b/Makefile.in	2023-06-20 11:07:14.611452052 -0700
@@ -129,8 +129,6 @@
 @ANDROID_HOST_TRUE@	-I$(top_srcdir)/src/common/android/testing/include
 
 # Build as PIC on Linux, for linux_client_unittest_shlib
-@LINUX_HOST_TRUE@am__append_2 = -fPIC
-@LINUX_HOST_TRUE@am__append_3 = -fPIC
 libexec_PROGRAMS = $(am__EXEEXT_10)
 bin_PROGRAMS = $(am__EXEEXT_2) $(am__EXEEXT_3) $(am__EXEEXT_4)
 check_PROGRAMS = src/common/safe_math_unittest$(EXEEXT) \
@@ -1744,7 +1742,7 @@
 HEADERS = $(includec_HEADERS) $(includecl_HEADERS) \
 	$(includeclc_HEADERS) $(includecldwc_HEADERS) \
 	$(includeclh_HEADERS) $(includeclm_HEADERS) \
-	$(includegbc_HEADERS) $(includelss_HEADERS) \
+	$(includegbc_HEADERS) \
 	$(includep_HEADERS)
 am__tagged_files = $(HEADERS) $(SOURCES) $(TAGS_FILES) $(LISP)
 # Read a list of newline-separated strings from the standard input,
@@ -9650,7 +9648,7 @@
 	install-includeclHEADERS install-includeclcHEADERS \
 	install-includecldwcHEADERS install-includeclhHEADERS \
 	install-includeclmHEADERS install-includegbcHEADERS \
-	install-includelssHEADERS install-includepHEADERS \
+	install-includepHEADERS \
 	install-pkgconfigDATA
 
 install-dvi: install-dvi-am
