diff -Naur a/folly/io/async/AsyncSocket.cpp b/folly/io/async/AsyncSocket.cpp
--- a/folly/io/async/AsyncSocket.cpp	2023-12-08 20:38:13
+++ b/folly/io/async/AsyncSocket.cpp	2023-12-14 13:46:14
@@ -18,6 +18,11 @@
 
 #include <sys/types.h>
 
+#if defined(__linux__)
+/* for struct sock_extended_err*/
+#include <linux/errqueue.h>
+#endif
+
 #include <cerrno>
 #include <climits>
 #include <sstream>
diff -Naur a/folly/io/async/AsyncUDPSocket.cpp b/folly/io/async/AsyncUDPSocket.cpp
--- a/folly/io/async/AsyncUDPSocket.cpp	2023-12-08 20:38:13
+++ b/folly/io/async/AsyncUDPSocket.cpp	2023-12-14 13:46:27
@@ -17,6 +17,11 @@
 #include <folly/io/SocketOptionMap.h>
 #include <folly/io/async/AsyncUDPSocket.h>
 
+#if defined(__linux__)
+/* for struct sock_extended_err*/
+#include <linux/errqueue.h>
+#endif
+
 #include <cerrno>
 
 #include <boost/preprocessor/control/if.hpp>
diff -Naur a/folly/net/NetOps.h b/folly/net/NetOps.h
--- a/folly/net/NetOps.h	2023-12-08 20:38:13
+++ b/folly/net/NetOps.h	2023-12-14 13:45:48
@@ -114,7 +114,7 @@
 #endif
 #endif
 /* for struct sock_extended_err*/
-#include <linux/errqueue.h>
+#include <linux/types.h>
 #endif
 #endif
 
